### PR TITLE
Add tests to render all admin pages

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: tests/App/Action/AdminsTest.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+        $this->prepareData();
+
+        $client->request('GET', $url);
+
+        self::assertResponseIsSuccessful();
+    }
+
+    /**
+     * @return \Generator<string[]>
```
Comment (VincentLanglet) [inline tests/App/Action/AdminsTest.php]: Shouldnt it be iterable instead of generator ?

**Review by VincentLanglet — DISMISSED** [review]

File: tests/App/Action/IntegrationTestCase.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+{
+    protected function setUp(): void
+    {
+        $kernel = self::createKernel();
+        $kernel->boot();
+
+        $application = new Application($kernel);
+        $application->setAutoExit(false);
+
+        $application->run(new ArrayInput([
```
Comment (jordisala1991) [inline tests/App/Action/IntegrationTestCase.php]: Would be nice to copy what has been done on doctrine orm admin bundle and mongodb admin bundle. To not introduce another way of doing this.

File: tests/App/Resources/config/config.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+use Symfony\Component\HttpFoundation\Session\Storage\MockFileSessionStorageFactory;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->extension('framework', ['test' => true]);
+
+    $containerConfigurator->extension('framework', ['secret' => 'MySecret']);
+
+    $containerConfigurator->extension('framework', ['form' => true]);
+
+    $containerConfigurator->extension('framework', ['translator' => true]);
```
Comment (jordisala1991) [inline tests/App/Resources/config/config.php]: I guess you dont need to split those that much, you could only do one call with the 4 lines

**Comment by core23** [comment]
I'm giving up on this (for now). Supporting symfony 4.4 to 6.0 is really annoying.

**Comment by jordisala1991** [comment]
> I'm giving up on this (for now). Supporting symfony 4.4 to 6.0 is really annoying.

I can continue the PR were you left, you are almost there

File: tests/App/Action/AdminsTest.php [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+        $client->request('GET', $url);
+
+        self::assertResponseIsSuccessful();
+    }
+
+    /**
+     * @return iterable<string[]>
+     *
+     * @phpstan-return Generator<array{string}>
```
Comment (VincentLanglet) [inline tests/App/Action/AdminsTest.php]: Iterable should be prefer too

**Review by VincentLanglet — DISMISSED** [review]

**Comment by jordisala1991** [comment]
Im trying to understand why it does not work with lowest (I tried locally with php 8.0 and it works), need more time to reproduce it with php7.4

**Comment by jordisala1991** [comment]
I think I found it.

**Comment by jordisala1991** [comment]
Well at least on my machine it works now with php 7.4 ... not sure why it does not on github actions, have to look more (not enough time right now, will look later)

**Comment by jordisala1991** [comment]
Now it works. Btw this bundle misses a lot of dependencies (at least framework bundle, but looking at composer json there are probably more)

**Review by jordisala1991 — APPROVED** [review]

**Comment by VincentLanglet** [comment]
> Now it works. Btw this bundle misses a lot of dependencies (at least framework bundle, but looking at composer json there are probably more)

Can you add the one you discovered as missing ?

**Comment by core23** [comment]
Thanks a lot @jordisala1991 ❤️

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**